console/templates/partials/submitter.html

```diff
@@ -1,32 +1,33 @@
-<div class='col-md-6'>
-  <form id='submitter-form'>
-    <h3>Submitter</h3>
+<div class="container">
+  <div class='col-md-6'>
+    <form id='submitter-form'>
+      <h3>Submitter</h3>
 
-    <div class="form-group">
-      <label for='survey-selector'>Load survey:</label>
-      <select id="survey-selector">
-        {% for survey in surveys %}
-          <option value='{{survey}}'>{{survey}}</option>
-        {% endfor %}
+      <div class="form-group">
+        <label for='survey-selector'>Load survey:</label>
+        <select id="survey-selector">
+          {% for survey in surveys %}
+            <option value='{{survey}}'>{{survey}}</option>
+          {% endfor %}
 
-      </select>
-    </div>
-    <br>
+        </select>
+      </div>
+      <br>
 
-    <div class="form-group">
-      <label for='post-data'>Data to send:</label>
-      <textarea id="post-data" class="input input--text" name='post-data' rows='10'></textarea>
-    </div>
-    <br>
+      <div class="form-group">
+        <label for='post-data'>Data to send:</label>
+        <textarea id="post-data" class="input input--text" name='post-data' rows='10'></textarea>
+      </div>
+      <br>
 
-    <div class="form-group">
-      <label for='survey-quantity'>Number of submissions:</label>
-      <input id="survey-quantity" class="input input--text" value="1" />
-    </div>
-    <br>
+      <div class="form-group">
+        <label for='survey-quantity'>Number of submissions:</label>
+        <input id="survey-quantity" class="input input--text" value="1" />
+      </div>
+      <br>
 
-    <button id="submit-data" class="btn btn-default">Submit</button>
-    <button id="validate" class="btn btn-default">Validate</button>
-  </form>
-</div>
-<br>
+      <button id="submit-data" class="btn btn-default">Submit</button>
+      <button id="validate" class="btn btn-default">Validate</button>
+    </form>
+  </div>
+</div>
```

Comment:
Really minor, but can you sort this out?  Files should have line endings on the last line unless theres a really good reason for it.